chrome/content/zotero/fileInterface.js

```diff
@@ -129,7 +131,9 @@ var Zotero_File_Interface = new function() {
 	 */
 	function exportFile() {
 		var exporter = new Zotero_File_Exporter();
-		exporter.name = Zotero.getString("pane.collections.library");
+		exporter.libraryID = ZoteroPane_Local.getSelectedLibraryID();
+		if exporter.libraryID === false throw('No library selected');
```

Comment:
Are you testing this? I'm fairly certain that's a syntax error. Can't test it right now, so in the off chance that it's not, please use parentheses for conditionals.